activator/steplib/activate.go

```diff
@@ -118,17 +54,9 @@ func Activate(stepLibURI, id, version, destination, destinationStepYML string, u
 	return nil
 }
 
-func queryStep(stepLib models.StepCollectionModel, stepLibURI string, id, version string, updateLibrary bool, log stepman.Logger) (models.StepModel, string, error) {
```

Comment:
`updateLibrary` was only called with `false` (except for a test), so I cleaned it up